food-truck-api/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.profiles.active=prod
```

Comment:
We should leave this as prod. In IntelliJ you can override it like this instead of changing the properties file:
![image](https://user-images.githubusercontent.com/6753860/96921938-d1969280-1474-11eb-8803-671e92af9fc4.png)